src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started transporting materials.");

```

Comment:
The message should be more informative and indicate that it is a Truck that is working. For example: 'Truck is working: started transporting materials.'